src/core/ext/filters/backend_metrics/backend_metric_filter.cc

```diff
@@ -92,4 +92,17 @@ ArenaPromise<ServerMetadataHandle> BackendMetricFilter::MakeCallPromise(
       }));
 }
 
+static bool maybe_prepend_grpc_backend_metric_filter(
```

Comment:
This can be a lambda, inlined on line 105.

Reply:
Done.